rest-api/src/main/resources/definitions/custom_activity_event_request.yml

```diff
@@ -0,0 +1,12 @@
+description: Request object for creating a Custom Activity Event.
+type: object
+required:
+    - eventKey
+properties:
+    eventKey:
```

Comment:
Returned by one object as eventId, specified here as eventKey. Also, I think we only need one object here... ? Request isn't that different from response.